CONTRIBUTING.md

```diff
@@ -14,3 +14,13 @@ poetry shell
 # install pre-commit hooks
 pre-commit install
 ```
+
+# Testing
+```
+# can be done using
+poetry run pytest
+
+# or in the poetry shell
+poetry shell
+pytest
+```
```

Comment:
Directly in the terminal:
```
poetry run pytest
```
In the Poetry shell, through the terminal:
```
poetry shell
pytest
```

Because of GitHub's way of leaving comments, I can't leave a suggestion with code blocks inside. Everything above is how I'd like to see these lines. Basically, it's about moving the talking out of the code block.